<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-1RN3IOUD" ignored="0" name="Flight Search Test" tests="2" failures="2" timestamp="12 Apr 2019 06:33:54 GMT" time="42.32" errors="0">
  <testcase name="Test1" time="26.926" classname="com.parallel.tests.HomePageTest1">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.parallel.tests.HomePageTest1.Test1(HomePageTest1.java:33)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- Test1 -->
  <testcase name="Test1" time="40.297" classname="com.parallel.tests.HomePageTest1">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.parallel.tests.HomePageTest1.Test1(HomePageTest1.java:33)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- Test1 -->
</testsuite> <!-- Flight Search Test -->
